### DISCUSSION TITLE
Does kwok create k8s node

### DISCUSSION THREAD

**Discussion by huiwq1990** [kubernetes-sigs/kwok#131] [Q&A] [answered]
As far as I know, kubelet is responsible for create kubernetes node, and old kwok repo has the related code https://github.com/wzshiming/fake-kubelet/blob/master/node_controller.go#L303. I can't find related codes in [kwok](https://github.com/kubernetes-sigs/kwok).

**Comment by wzshiming** [comment, marked as answer]
In Kwok you just need to create the node you want directly, Kwok is no longer responsible for creating nodes.

https://github.com/kubernetes-sigs/kwok/blob/v0.0.1/docs/examples/kwok-manage-nodes-and-pods.md#create-a-node
